nncf/__init__.py

```diff
@@ -42,6 +42,9 @@
 from nncf.quantization import compress_weights as compress_weights
 from nncf.quantization import quantize as quantize
 from nncf.quantization import quantize_with_accuracy_control as quantize_with_accuracy_control
+from nncf.quantization.advanced_parameters import (
+    AdvancedAccuracyRestorerParameters as AdvancedAccuracyRestorerParameters,
+)
```

Comment:
Shouldn't we introduce an example with these parameters then?

Reply:
Thanks for the comment. The ticket 131817 was already filed.

Reply:
Great!